### ISSUE TITLE
Low contrast and other accessibility warnings

### ISSUE THREAD

**Issue by openmindculture** [openmindculture/ingo-steinke.de#176]
18 total issues in axe DevTools: elements must meet minimum color contrast ratio thresholds:
- .brand-orange.hashtag x6
- .brand-blue-liquid.hashtag x4
- .brand-red-light.hashtag x5
- .brand-blue-php.hashtag x3

3 contrast errors in WAVE: Very low contrast (3.04:1)
- .brand-blue-php.hashtag x3

5 alerts in WAVE:
- 1x long alternative text (false positive: quoting text contained in image)
- 1x possible heading: tagline below H1 (no benefit in creating another heading, that would break the text flow when read aloud)
- 1x broken same-page link: "und für WordPress": `href="#project-wordpress-plugin"`
  ^ correct! that's been removed during project list content update. To do: removing the link would be fine!
- 2x redundant title text: title attribute text is the same as text or alternative text: .tile .code-link
  ^ correct! title should be removed
